pkgs/applications/display-managers/ly/default.nix

```diff
@@ -23,7 +24,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
```

Comment:
Do you think we could add the `platforms` meta attribute?

Reply:
Depends on which arch the derivation is working with.

Reply:
Thanks, without `platforms` it nixpkgs-review didn't even start to build on `aarch64-linux`, but now it does (successfully).

Reply:
This is not only for that, but also to display the proper platforms it runs on on search.nixos.org ;)

Reply:
According to https://github.com/fairyglade/ly it should work on "Linux and BSD", so I think it's good now.